src/lib/tabs/tab-nav-bar/tab-nav-bar.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, Input, ViewChild, ElementRef } from '@angular/core';
+import { MdInkBar } from '../ink-bar';
+
+@Component({
+  moduleId: module.id,
+  selector: 'md-tab-nav-bar',
+  templateUrl: 'tab-nav-bar.html',
+  styleUrls: ['tab-nav-bar.css'],
+})
+export class MdTabNavBar {
+  @ViewChild(MdInkBar) private _inkBar: MdInkBar;
```

Comment:
`@ViewChild` can't be applied to a private member (since Angular itself is external to this class and has to touch it).